Source/Classes/Internal/OverlayContainer/BackgroundAndOverlayViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  BackgroundAndOverlayViewController.swift
+//  DynamicOverlay
+//
+//  Created by Ivo Silva on 05.03.21.
+//  Copyright © 2021 Fabernovel. All rights reserved.
+//
+
+import SwiftUI
+import OverlayContainer
+
+class BackgroundAndOverlayContainerViewController: OverlayContainerViewController {
```

Comment:
Did you try to use the `OverlayContainer.viewControllers` property? The first VC will be the background, the last one the overlay.